darktheme.css

```diff
@@ -0,0 +1,112 @@
+@import url(alert/alert-dark.css);
+@import url(avatar/avatar.css);
+@import url(image/image.css);
+@import url(button/button.css);
+/* @import url(badge/badge.css);
+@import url(card/card.css); */
+
+/* ------------- */
+/* Setting up the custom properties */
+/* ------------- */
+
+
+html {
+    box-sizing: border-box;
+}
+
+
+:root {
+
+    --color-primary: rgb(145, 128, 196);
+    --color-primary-values: 145, 128, 196;
+
+    --color-white: rgb(247, 247, 247);
+    --color-dark: rgb(11, 13, 23);
+    --color-secondary: rgb(156, 163, 175);
+
+    --color-complementary: rgb(247, 247, 247);
+    --color-complementary-values: 247, 247, 247;
+
+    --color-current: rgb(11, 13, 23);
+    --box-shadow: 0 14px 8px -2px rgba(255, 255, 255, 0.3);
+
+    /* font-size */
+    --fsize-700: 4rem;
+    --fsize-600: 3rem;
+    --fsize-500: 2rem;
+    --fsize-400: 1.5rem;
+    --fsize-300: 1.25rem;
+    --fsize-200: 1rem;
+    --fsize-100: 0.875rem;
```

Comment:
This is very convenient. You defined all useable font size one place. Awesome approach.

Reply:
Thank you